servers/camera_server.h

```diff
@@ -59,6 +59,7 @@ class CameraServer : public Object {
 	};
 
 	typedef CameraServer *(*CreateFunc)();
+	static inline constexpr const char camera_feeds_updated[] = "feeds_updated";
```

Comment:
I also meant to update the string value to `camera_feeds_updated` since in code a user would do:
```
CameraServer.camera_feed_added.connect(...)
CameraServer.camera_feed_removed.connect(...)
CameraServer.camera_feeds_updated.connect(...)
```

Reply:
Fixed the signal name. Also reverted the variable name back to the original.